src/pages/docs/actions.mdx

```diff
@@ -76,3 +76,49 @@ You can retrieve a registered UI schema via the `findUISchema` function.
 
 This action allows to unregister a previously registered UI schema
 
+## `findUISchema(uischemas: object[], schema: JsonSchema, schemaPath: string, path: string, fallbackLayoutType?: string, control?: ControlElement, rootSchema?: JsonSchema)`
```

Comment:
This should not be an own section but still be a part of the previous section. This is just a utility function and not one of the available actions.